sqlest/src/test/scala/sqlest/executor/ExecutorSpec.scala

```diff
@@ -145,4 +153,12 @@ class ExecutorSpec extends FlatSpec with Matchers {
       optionInsertStatement.execute
     }
   }
+
+  "an insert with mapped optional values" should "execute correctly" in {
+    implicit val testDatabase = TestDatabase(testResultSet)
+
+    testDatabase.withTransaction {
+      mappedOptionInsertStatement.execute
```

Comment:
This doesn't actually check that the mapping is being done. I realise that none of the other tests in this unit currently check the arguments set on the statement but maybe this is a good time to start